easybuild/easyconfigs/b/Biopython/Biopython-1.79-foss-2021b.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'PythonPackage'
+
+name = 'Biopython'
+version = '1.79'
+
+homepage = 'https://www.biopython.org'
+description = """Biopython is a set of freely available tools for biological
+ computation written in Python by an international team of developers. It is
+ a distributed collaborative effort to develop Python libraries and
+ applications which address the needs of current and future work in
+ bioinformatics. """
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+source_urls = ['https://biopython.org/DIST']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['edb07eac99d3b8abd7ba56ff4bedec9263f76dfc3c3f450e7d2e2bcdecf8559b']
+
+dependencies = [
+    ('Python', '3.9.6', '-bare'),
```

Comment:
@sassy-crick the `-bare` python can make sense as a builddep, but usually not as a dependency and certainly not together with `SciPy-bundle`...

Reply:
@migueldiascosta That makes sense. I changed that. Thanks!